subprojects/parseq/src/test/java/com/linkedin/parseq/TestTaskStates.java

```diff
@@ -268,8 +268,8 @@ private void runTask(final Task<?> task) {
   private void assertInitOrScheduled(final Task<?> task) {
     assertFalse(task.isDone());
     assertFalse(task.isFailed());
-    assertNull(task.getShallowTrace().getStartNanos());
-    assertNull(task.getShallowTrace().getEndNanos());
+    assertEquals(task.getShallowTrace().getNativeStartNanos(), -1);
```

Comment:
We should create new tests for new APIs to make sure that our change will not break existing users.

Reply:
to keep it clean, I had tested these Uts with deprecated methods. added new tests for deprecated methods separately. Check TestShallowTraceBuilder#testDeprecatedTimeMethods